.github/ISSUE_TEMPLATE/improve-existing-docs.yml

```diff
@@ -0,0 +1,62 @@
+name: Improve existing docs
+about: Make a suggestion to improve our existing documentation.
+labels: content
+inputs:
+  - type: description
+    attributes:
+      value: |
+        HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
+
+        For questions, ask in Discussions: https://github.com/github/docs/discussions
+  - type: dropdown
+    label: Code of Conduct
+    attributes:
+      choices:
+        - description: 'I agree to follow the Code of Conduct: https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md'
+          required: true
+          multi-select: false
+          free-text: false
+          choices:
+            - description: I agree
+              value: I agree
+  - type: dropdown
+    label: Contributing
+    attributes:
+      choices:
+        - description: 'I have read the Contributing guide: https://github.com/github/docs/blob/main/CONTRIBUTING.md'
+          required: true
+          multi-select: false
+          free-text: false
+          choices:
+            - description: I agree
+              value: I agree
+  - type: dropdown
+    label: Similar issue
+    attributes:
+      choices:
+        - description: 'There is not a similar issue that has been already opened: https://github.com/github/docs/issues'
+          required: true
+          multi-select: false
+          free-text: false
+          choices:
+            - description: No similar issues
+              value: No similar issues
+  - type: textarea
+    attributes:
+      label: which article
+      description: What article on docs.github.com is affected?
+      required: true
+      placeholder: Add a link to the article you'd like to see updated
```

Comment:
(Keeping in mind I'm not sure how this will render) What if there's more than one article? Will there be space for a list?

Reply:
Here's a sample of how things should render? 

https://github.com/github/docs-content/issues/new?assignees=&labels=Improve+existing+docs%2C+triage&template=improve-existing-docs.yml&title=%5BImprovement%5D%3A+

Reply:
So, I think a list should work.